lib/java/test/org/apache/thrift/test/TestServer.java

```diff
@@ -190,7 +193,9 @@ public static void main(String [] args) {
         if (protocol_type.equals("binary")) {
         } else if (protocol_type.equals("compact")) {
         } else if (protocol_type.equals("json")) {
-        } else if (protocol_type.equals("multiplexed")) {
+        } else if (protocol_type.equals("multi")) {
```

Comment:
I did it this way to work within the spec:impl naming convention that currently exists in make cross.  See tests.json, specifically "binary:accel" or "compact:accelc".  I wanted to follow the pattern that already existed in the test suite so we have a single use pattern, not two.  In the end the behavior is mostly the same, except by using "multi:binary" on the java server and "binary:multi" on the c_glib client, we end up testing:  c_glib (binary client) => java (multi server) as well as c_glib (multi client wrapping binary) => java (multi server wrapping binary).  Using the "multiplexed-binary" naming convention would not have leveraged the existing logic in crosstest/collect.py to make this happen.